docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+version: "3"
+services:
+  db:
+    ports:
+      - 55432:5432
+    image: registry.access.redhat.com/rhscl/postgresql-10-rhel7
+    environment:
+      - POSTGRESQL_USER=user
+      - POSTGRESQL_PASSWORD=password
+      - POSTGRESQL_ADMIN_PASSWORD=admin
+      - POSTGRESQL_DATABASE=aggregator
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+      - ZOOKEEPER_LOG4J_ROOT_LOGLEVEL=ERROR
+      - ZOOKEEPER_TOOLS_LOG4J_LOGLEVEL=ERROR
+  kafka:
+    image: confluentinc/cp-kafka
+    ports:
+      - 29092:29092
+    depends_on:
+      - zookeeper
+    environment:
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:29092
+      - KAFKA_BROKER_ID=1
+      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:32181
+  minio:
+    image: minio/minio
+    command: 
+      - server
+      - /data
+    volumes:
+      - ../minio/data/:/data:Z
+      - ../minio/config:/root/.minio:Z
+    ports:
+      - 9000:9000
+    environment:
+      - MINIO_ACCESS_KEY=BQA2GEXO711FVBVXDWKM
+      - MINIO_SECRET_KEY=uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC
+  createbuckets:
+    image: minio/mc
+    depends_on:
+      - minio
+    entrypoint:
+      - /bin/sh
+      - -c
+      - "sleep 5;
+      /usr/bin/mc config host add myminio http://minio:9000 BQA2GEXO711FVBVXDWKM uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC;
```

Comment:
that always confuses me, as mc is Midnight Commander on my machine. But of course, minio client is perfectly ok inside docker :)